jquery.lazyload.js

```diff
@@ -126,6 +126,9 @@
                                 var elements_left = elements.length;
                                 settings.load.call(self, elements_left, settings);
                             }
+                        }).bind("error", function() {
+                            var fallback = $self.attr("data-fallback");
+                            $self.attr("src", fallback);
```

Comment:
Does anything weird happen if data-fallback isn't present?
Would it be safer to only set src if there is data-fallback?